yeat/cli.py

```diff
@@ -119,6 +131,14 @@ def get_parser():
         default=0,
         help="downsample reads down to the desired number; by default, D=0; when set to default, YEAT will auto downsample; set D=-1 to not downsample",
     )
+    parser.add_argument(
+        "-c",
+        "--coverage",
+        type=check_positive,
```

Comment:
This is where the `check_positive` is being called.

To be honest, we can solve the throw `ValueError` problem above if I changed this to an `int` instead of `check_positive`.

In doing so, in order to check if the `--coverage` is <= 0, I would need to check after the CLI is parsed (probably in the snakemake). Instead, I followed the parse_args option by assigning the flag input `type` with a function.